nginx.conf

```diff
@@ -27,6 +27,8 @@ http {
   server {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
+	
+	client_max_body_size 40M; #increase upload size limit
```

Comment:
oops, I didn't mean to have the indent there. I can remove on my next PR for redirects.